.github/workflows/build.yml

```diff
@@ -427,6 +427,57 @@ jobs:
             library-x86_64-linux-android
           failOnError: false
 
+  alpha-release:
+    name: Alpha Release
+    if: github.event_name == 'push'
```

Comment:
I think this might need to check the branch name is `main` as well?

Reply:
on line 9 it only listens for the push event on main, so this job will only run on main. But I do think it's a good idea to add an extra check here in case the event listener up top changes in the future